src/diffusers/training_utils.py

```diff
@@ -24,6 +24,9 @@
 if is_transformers_available():
     import transformers
 
+    if transformers.deepspeed.is_deepspeed_zero3_enabled():
```

Comment:
`transformers.deepspeed` is deprecated and will be removed in a near future release. Could you look into the alternative `transformers.integrations`?

Reply:
Yes, give me sometime.